test_workflows.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# if GALAXY_URL and GALAXY_USER_KEY are not set, try to set them from local untracked file
+[ ! $GALAXY_URL ] || [ ! $GALAXY_USER_KEY ] && source .secret.env
+export GALAXY_URL GALAXY_USER_KEY
+
+virtualenv -p python3 .venv; . .venv/bin/activate
```

Comment:
On jenkins this is done through a plugin that creates a venv (shining panda), so this step isn't necessary. But the pip install is useful.

Reply:
Probably best to `pip install -r requirements.txt` within code on jenkins